Geometry/EcalCommonData/plugins/dd4hep/DDEcalBarrelNewAlgo.cc

```diff
@@ -2144,27 +2106,29 @@ static long algorithm(dd4hep::Detector& /* description */, cms::DDParsingContext
       Solid backSideSolid = mytrap(back.sideName, trapBS);
       Volume backSideLog = Volume(back.sideName, backSideSolid, ns.material(back.sideMat));
 
-      const Position backSideTra1(0, back.plateWidth / 2 + back.sideYOff1, 1 * dd4hep::mm);
+      const Position backSideTra1(0 * dd4hep::mm, back.plateWidth / 2. + back.sideYOff1, 1 * dd4hep::mm);
```

Comment:
Here I would use "0" as well, if it is changed from "/2" to "/2." I would use "*0.5", unfortunately the effect of multiplication may be seen if it is applied everywhere, not in a single line.

Reply:
@civanch - I've changed the division by the multiplication. Thanks!